### PR TITLE
Update for new Makie

### PR DESCRIPTION
**Pull request by asinghvi17**
This updates some of the example code for Makie.jl v0.19.  It runs pretty well on my machine!

https://user-images.githubusercontent.com/32143268/227084853-780138e9-b850-4c02-9365-6621db1bc823.mp4

### CODE REVIEW

**Comment by weymouth** [comment]
This has a lot of bugs, and doesn't even compile on my system. I've fixed it and tested it on my system, but I don't understand how it worked on yours...

File: examples/ThreeD_Plots.jl [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 
-function volume_video!(sim,data_func;name="file.mp4",duration=1,step=0.1; framerate = 30, compression = 20)
+function volume_video!(sim,data_func;name="file.mp4",duration=1,step=0.1,framerate = 30, compression = 20)
     # Set up viz data and figure
     dat = Observable(data_func(sim))
     fig, ax, plt = volume(
         dat,
         colorrange=(π,4π),
-        algorithm=:absorption,
-        axis = (backgroundcolor = RGBf0(0.9, 0.9, 0.9),)
```
Comment (asinghvi17) [inline examples/ThreeD_Plots.jl]: No reason this line shouldn't have worked - what was your Makie version?

File: examples/ThreeD_Plots.jl [hunk 2]
```diff
@@ -21,7 +19,7 @@ function volume_video!(sim,data_func;name="file.mp4",duration=1,step=0.1; framer
     return sim,fig
 end
 
-function contour_video!(sim,data_func,geom_func;name="file.mp4",duration=1,step=0.1; framerate = 30, compression = 20)
```
Comment (asinghvi17) [inline examples/ThreeD_Plots.jl]: oops, sorry about these - I guess I hadn't committed the typo fixes.

**Review by asinghvi17 — COMMENTED** [review]
Hey, thanks for the bugfix - I think I accidentally didn't commit the typo fixes.

File: examples/ThreeD_Plots.jl (hunk 1 above)
Comment (asinghvi17) [reply examples/ThreeD_Plots.jl]: Yeah, I figured...that keyword probably wouldn't work for `LScene`.

**Pull request merged**